### DISCUSSION TITLE
EP resets during normal operations - requires uninstall/reinstall

### DISCUSSION THREAD

**Discussion by WillMartin58** [valinet/ExplorerPatcher#1644] [Q&A] [answered]
Folks,

Granted, I'm on an insider version of Windows (23493.ni_prerelease.230624-1828), but after the last round of patching, after I restart my computer Explorer (or Explorer Patcher, but Explorer is what flares the event log) dies and restarts over and over.  If I uninstall EP (build 22621.1555.55.1) and reinstall it, this stops until my next restart.  Anyone else seen this issue, and (more importantly) know how I can remedy it without the uninstall?

**Comment by pyrates999** [comment, marked as answer]
Insider builds and preview updates are not supported due to debug symbols not being available to download, and explorer patcher requires them to function.

So check if symbols were downloaded: https://github.com/valinet/ExplorerPatcher/wiki/Symbols

You can check using the console in explorer patcher.  Open up the properties application for explorer patcher, go to advanced, and click on enable console.  If it shows it can't download symbols, your version of windows is not supported.

**Comment by WillMartin58** [comment]
I assumed as much.  I can uninstall and reinstall as needed.  Thanks.  🙂
